src/distilabel/utils/imports.py

```diff
@@ -115,3 +115,4 @@ def _check_package_is_available(
     "google-generativeai", min_version="0.3.2", greater_or_equal=True
 )
 _TOGETHER_AVAILABLE = _check_package_is_available("together")
+_MISTRALAI_AVAILABLE = _check_package_is_available("mistralai")
```

Comment:
versioning? Perhaps it is better to not do it but just to check.

Reply:
Will make it check the same version from pyproject.toml